code/datums/actions/items/toggles.dm

```diff
@@ -38,6 +38,12 @@
 /datum/action/item_action/toggle_welding_screen
 	name = "Toggle Welding Screen"
 
+/datum/action/item_action/blow_whistle
+	name = "Blow Your Whistle"
+
+/datum/action/item_action/wrist_strap
+	name = "Adjust Wrist Strap"
+	
```

Comment:
if you want my opinion on it just move this to the files where the actions is actually run, i.e the whistle file and the baton one just above or below the item and their procs